HW_3/src/main/java/com/homework3/hw_3/controller/LinerController.java

```diff
@@ -0,0 +1,51 @@
+package com.homework3.hw_3.controller;
+
+import com.homework3.hw_3.entity.Liner;
+import com.homework3.hw_3.entity.dto.LinerDTO;
+import com.homework3.hw_3.service.inter.LinerService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController(value = "/liner")
+@RequiredArgsConstructor
+public class LinerController {
+    private final LinerService linerService;
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/{id}")
+    public LinerDTO getLinerById(@PathVariable long id) {
+        log.info("get liner by id {}", id);
+        return linerService.getLinerById(id);
+    }
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/")
```

Comment:
Don't understand why these "/" are needed, try without them. If don't work, then try to use @RequestMapping("/liner") and just @RestController